include/ultra64.h

```diff
@@ -8,18 +8,22 @@
 
 #include <ultra64/types.h>
 #include <ultra64/interrupt.h>
-#include <ultra64/os_misc.h>
 #include <ultra64/rcp.h>
 #include <ultra64/rdp.h>
 #include <ultra64/rsp.h>
-#include <ultra64/os_thread.h>
+#include <ultra64/thread.h>
 #include <ultra64/convert.h>
 #include <ultra64/time.h>
-#include <ultra64/os_message.h>
+#include <ultra64/message.h>
 #include <ultra64/sptask.h>
 #include <ultra64/gu.h>
-#include <ultra64/os_vi.h>
-#include <ultra64/os_pi.h>
-#include <ultra64/os_internal.h>
+#include <ultra64/vi.h>
+#include <ultra64/pi.h>
+#include <ultra64/internal.h>
```

Comment:
Internal libultra stuff is not supposed to be part of `ultra64.h`, since it's not part of the external interface. I think it should probably stay included only where needed.